packages/runtime/container-runtime/src/opLifecycle/opCompressor.ts

```diff
@@ -48,7 +48,16 @@ export class OpCompressor {
 		});
 
 		for (const message of batch.content.slice(1)) {
-			messages.push({ ...message, contents: undefined });
+			// Add an empty placeholder message to reserve the sequence numbers
+			messages.push({
+				deserializedContent: {
```

Comment:
It's good change, but I do not fully understand how this change matters / how deserializedContent gets up to socket.
Looking at sendBatch(), it will drop deserializedContent property. So it feels like it should never get to socket, no?

Reply:
FYI - I've opened https://dev.azure.com/fluidframework/internal/_queries/edit/3452/?triage=true earlier - ideally we do not even keep around deserialized state.